private/testeth/gen.go

```diff
@@ -0,0 +1,7 @@
+// Copyright (C) 2022 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package testeth
+
+//go:generate mkdir -p "testtoken"
+//go:generate abigen --bin=../../contracts/build/TestToken.bin --abi=../../contracts/build/TestToken.abi --type=TestToken --pkg=testtoken --out=testtoken/token.go
```

Comment:
Whenever I try a go generate, I get the error
`Fatal: Failed to read input ABI: open ../../contracts/build/TestToken.abi: The system cannot find the path specified.`
What am I doing wrong? I've tried running `go install` and `go build` but I do not see the `/contracts/build` folder showing up anywhere.

Reply:
You need to compile contract first. Pls look at the PR with contracts/compile.sh.
Also, build folder is ignored as there is no need to push compiled output to the repo